Add go mod files

anacrolix opened the pull request:
Fixes #1. Also fixes the line `go: finding github.com/op/go-logging latest` always coming up.

anacrolix committed:
Add go mod files

elgatito commented:
@anacrolix , thanks. I don't use this package for a long time, so maybe it would be better to make it readonly and link to your fork? :)

anacrolix commented:
Okay, that sounds good. Cheers

anacrolix committed:
Suppress logging

For now I've just implemented the expected logger interface for fixing later. The API to this package is all wrong anyway.

anacrolix committed:
gofmt

anacrolix committed:
Switch to anacrolix/log and remove package-level logging

anacrolix committed:
Change module name elgatito->anacrolix

anacrolix committed:
Update anacrolix/log

anacrolix commented:
See https://github.com/anacrolix/upnp instead.
